src/biogeochem/CNFireBaseMod.F90

```diff
@@ -881,21 +1068,37 @@ subroutine CNFireFluxes (this, bounds, num_soilc, filter_soilc, num_soilp, filte
      ! vertically-resolved decomposing C/N fire loss   
      ! column loop
      !
+     num_actfirec = 0
      do fc = 1,num_soilc
         c = filter_soilc(fc)
 
         f = farea_burned(c) 
 
+        if(f .ne. 0 .or. f .ne. baf_crop(c))then
```

Comment:
For consistent formatting please use /= instead of .ne. . Please make this similar change in the other parts of the added code.